changes/5687.housekeeping

```diff
@@ -0,0 +1 @@
+Added structlog.
```

Comment:
`.housekeeping` is for non-user-facing changes such as refactoring. Since this changes the log format substantially, this should be a `.changed` note instead, and probably also should have a `.dependencies` note to indicate the new project dependency.

Reply:
Fixed by adding other fragments.